ofctrl/ofAction.go

```diff
@@ -58,6 +59,18 @@ type OFAction interface {
 	GetActionType() string
 }
 
+type PushVLANAction struct {
```

Comment:
Did you try `SetVLANAction`? What is the difference on the packet from this new action?